README.md

```diff
@@ -158,57 +160,9 @@ The `/static` and `/template` directories are replicated in `out/`, and the
 files in `src/` are rebased into `out/` (so `src/base/foo.py` becomes
 `out/base/foo.py`).
 
-
-## Detailed Dependency Information
--------------
-* The AppEngine SDK should be present in the directory:
```

Comment:
Some of it is, let me take another look, I guess I was trying to reduce the size of the readme to make it a little more digestible.